build/README.md

```diff
@@ -339,11 +346,46 @@ Now you're ready to begin the development/test cycle:
 
 You can combine some of the above steps into a single one, for example `make build push install` or `make build push test-e2e`.
 
-If you need to clean-up your cluster, you can use `make uninstall` to remove Agones and `make clean-custom-test-cluster` to reset helm.
+If you need to clean up your cluster, you can use `make uninstall` to remove Agones and 
+`make clean-custom-test-cluster` to delete the entire cluster.
```

Comment:
I don't see the target `clean-custom-test-cluster` defined anywhere in our repository. The documentation previously said it reset helm and you wrote that it deletes the cluster, but have you tested it to see if it works for you?

```bash
make: *** No rule to make target 'clean-custom-test-cluster'.  Stop.
```